themes/bootstrap3/templates/RecordDriver/DefaultRecord/data-summary.phtml

```diff
@@ -1,10 +1,18 @@
 <?php foreach ($this->driver->getSummary() as $summary): ?>
-  <?=$this->escapeHtml($summary) ?><br>
+  <?=$this->escapeHtmlExt($summary, allowHtml: true) ?><br>
 <?php endforeach; ?>
 <?php $isbn = $this->driver->getCleanISBN(); ?>
 <?php foreach ($this->summaries($isbn) as $provider => $content): ?>
-  <?php foreach ($content as $summary): ?>
-    <?php $htmlContent = str_starts_with($summary, '<') && str_ends_with($summary, '>'); ?>
-    <?=$htmlContent ? $summary : ($this->escapeHtml($summary) . '<br>')?>
-  <?php endforeach; ?>
+  <?=
+    implode(
+        '<br>',
+        array_map(
+            function ($summary) {
+                $htmlContent = str_starts_with($summary, '<') && str_ends_with($summary, '>');
```

Comment:
I wonder if there's a way to eliminate this hacky check for angle brackets. Should we do the check in the record driver and wrap the string in a PropertyString? Should the new helper have a switch to enable this check, or even make this behavior part of the standard "allowHtml"? I'm not totally sure, but it feels like we can do better here, especially since as written, there's no safety on the summary in the special case.